packages/web-react/src/components/ButtonLink/ButtonLink.tsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import classNames from 'classnames';
+import { WithChildren } from '../../types/main';
+import { compose } from '../../utils/compose';
+import { applyColor, applyClassNamePrefix } from '../../utils/classname';
+import { useClassNamePrefix } from '../../hooks/useClassNamePrefix';
+
+type Color = 'primary' | 'secondary' | 'tertiary';
+
+export interface ButtonLinkProps extends WithChildren {
+  color: Color;
+  href: string;
+  target?: string;
+  block: boolean;
+  disabled: boolean;
+  ariaLabel?: string;
+  componentClassName: string;
+  className?: string;
+}
+
+const getButtonColorClassname = (componentClassName: string, color: Color): string =>
+  compose(applyColor<Color>(color))(componentClassName);
+
+export const ButtonLink = ({
+  color,
+  href,
+  target,
+  block,
+  ariaLabel,
+  disabled,
+  className,
+  componentClassName,
+  ...restAttributes
+}: ButtonLinkProps): JSX.Element => {
+  const classNamePrefix = useClassNamePrefix();
+  const mainClassName = applyClassNamePrefix(classNamePrefix)(componentClassName);
+  const buttonBlockClassName = applyClassNamePrefix(classNamePrefix)('Button--block');
```

Comment:
@janicekt 👀 https://github.com/lmc-eu/spirit-design-system/blob/main/packages/web-react/src/components/Button/Button.tsx#L48

I think that it will be better when prefix is applied once and then we should use template literals for derived classes.